src/v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -59,7 +60,20 @@ const getConversions = (message, metadata, { Config }, event, conversionType) =>
     payload = constructPayload(message, trackCallConversionsMapping);
     endpoint = CALL_CONVERSION.replace(':customerId', filteredCustomerId);
   }
-
+  const consentObject = populateConsentForGoogleDestinations(message, conversionType);
+  if (Object.keys(consentObject)?.length > 0) {
+    if (payload?.conversions?.length > 0) {
+      if (conversionType === 'click' || conversionType === 'call') {
+        payload.conversions[0].consent = consentObject;
+      }
+    } else if (
+      Object.keys(payload?.addConversionPayload?.operations?.create)?.length > 0 &&
+      conversionType === 'store'
```

Comment:
Swap the order of the conditions. The string comparison is a less costly operation, so it should be the first one in an "AND" logic.

Reply:
removed